snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
Ouch, missed the 'd' in sshd. Still, ssh-keygen can specifcy protocols, bits, and output files - presumably it could be called appropriately? Something like `ssh-keygen -b 2048 -t rsa -f /etc/ssh/ssh_host_rsa_key` maybe?